dashboard/src/components/dialogs/binding/node-binding-dialog.ts

```diff
@@ -201,21 +201,40 @@ export class NodeBindingDialog extends LitElement {
   }
 
   async addBindingHandler() {
-    const targetNodeId = parseInt(this._targetNodeId.value, 10);
-    const targetEndpoint = parseInt(this._targetEndpoint.value, 10);
-    const targetCluster = parseInt(this._targetCluster.value, 10);
-
-    if (isNaN(targetNodeId) || targetNodeId <= 0) {
+    const targetNodeId = this._targetNodeId.value
+      ? parseInt(this._targetNodeId.value, 10)
+      : undefined;
+    const targetEndpoint = this._targetEndpoint.value
+      ? parseInt(this._targetEndpoint.value, 10)
+      : undefined;
+    const targetCluster = this._targetCluster.value
+      ? parseInt(this._targetCluster.value, 10)
+      : undefined;
+
+    if (
```

Comment:
```suggestion
    // Matter Server does not use random NodeIds, so this is ok for now, but needs to be adjusted later
    if (
```

Reply:
NodeIds are officially 64bit numbers, so ideally the textfield later should be a text and then you convert to BigInt. For now that works. When switching to matter.js finally I would like to go away from increasing node ids